src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterLaunchSpecRepo.java

```diff
@@ -0,0 +1,97 @@
+package com.spotinst.sdkjava.model;
```

Comment:
Create a 'repo' folder under model and move this class and its base class to repo folder under correct hierarchy.